tests/distributed_grids/large_vtu_01.with_p4est=true.with_zlib=on.mpirun=4.output

```diff
@@ -7,6 +7,8 @@ DEAL:0::<?xml version="1.0" ?>
 <Piece NumberOfPoints="216" NumberOfCells="64" >
 OK
 
-DEAL:1::n_global_active_cells: 8 n_global_levels: 2
-DEAL:1::OK
```

Comment:
@tjhei You had output from two of the four participating MPI ranks. I got output from all of the participating MPI ranks.

Reply:
Weird. Did I forget to update my output or is something broken?!

Reply:
That might have been the case - we do not run all configurations in the CI.